AABB_tree/demo/AABB_tree/CMakeLists.txt

```diff
@@ -17,58 +17,55 @@ if(POLICY CMP0071)
 endif()
 
 # Include this package's headers first
-include_directories( BEFORE ./ ./include )
+include_directories(BEFORE ./ ./include)
 
 # Find CGAL and CGAL Qt5
-find_package(CGAL COMPONENTS Qt5)
+find_package(CGAL REQUIRED COMPONENTS Qt5)
```

Comment:
If you add `REQUIRED` before `COMPONENTS Qt5`, then what happens if CGAL is found but not `CGAL_Qt5`?

Reply:
The doc statse "A package-specific list of required components may be listed after the COMPONENTS option (or after the REQUIRED option if present). Additional optional components may be listed after OPTIONAL_COMPONENTS. Available components and their influence on whether a package is considered to be found are defined by the target package."
So as I understand, if Qt5 is not found, there is a CMake error.

Reply:
Then you must remove the `REQUIRED`, here. And re-add the test on `CGAL_FOUND`.

Because Qt5 is not a required dependency for CGAL and, for the CTest testsuite to work correctly, the whole configuration of CGAL (with `-DWITH_tests=ON -DWITH_examples=ON -DWITH_demos=ON`) must finish without errors.

Actually, you should verify. I do not know if our `CGALConfig.cmake` files have a correct behavior with `REQUIRED` and with components.

Reply:
I think you are wrong. I had to make all the CGAL components optional instead, and keep the tests as they were. I checked, an OPTIONAL_COMPONENT works as expected even with a REQUIRED CGAL, as stated in the piece of doc I pasted above.